vgen/src/test/java/io/gapi/vgen/testdata/python_no_path_templates.baseline

```diff
@@ -98,18 +89,20 @@ class NoTemplatesServiceApi(object):
         """
         if scopes is None:
             scopes = self._ALL_SCOPES
-        bundling_override = bundling_override or {}
-        retrying_override = retrying_override or {}
-        client_config = pkg_resources.resource_string(
-            __name__, 'no_templates_service_client_config.json')
-        self._defaults = api_callable.construct_settings(
-            'google.example.v1.NoTemplatesService',
-            json.loads(client_config),
-            bundling_override,
-            retrying_override,
-            config.STATUS_CODE_NAMES,
-            timeout
-        )
+        if isinstance(client_config, google.gax.CallSettings):
+            self._defaults = client_config
+        else:
+            if not client_config:
+                client_config = json.loads(pkg_resources.resource_string(
+                    __name__, 'no_templates_service_client_config.json'))
+            self._defaults = api_callable.construct_settings(
```

Comment:
What if we do

```
client_config = json.loads(pkg_resources.resource_string(__name__, 'no_templates_service_client_config.json'))
client_config.update(user_client_config)
```

Then the user only needs to specify the options for the methods they are overriding. (Even better, do the update in the method dictionaries, so that you specify a bundling options and not override the retry options?)